lib/NfcoreTemplate.groovy

```diff
@@ -61,7 +80,7 @@ class NfcoreTemplate {
         misc_fields['Nextflow Compile Timestamp'] = workflow.nextflow.timestamp
 
         def email_fields = [:]
-        email_fields['version']      = workflow.manifest.version
+        email_fields['version']      = NfcoreTemplate.version(workflow)
```

Comment:
You probably don't need the `NfcoreTemplate` bit since its in the same file

Reply:
this was an nf-core TEMPLATE change